ERCS/erc-7390.md

```diff
@@ -0,0 +1,364 @@
+---
+eip: 7390
+title: Vanilla Option Standard
```

Comment:
You will have to remove "standard" from your title (as indicated by the bot.) Every ERC standardizes something, so calling an ERC a standard provides no useful information.